src/tests/ie_test_utils/common_test_utils/ngraph_test_utils.cpp

```diff
@@ -35,7 +35,7 @@ TransformationTestsF::TransformationTestsF()
     : model(function),
```

Comment:
we need to migrate utils to new namespace
Also, will be good to defer files to `include` and `src` dir